include/fixed_containers/reflection.hpp

```diff
@@ -225,25 +242,23 @@ template <typename T>
     requires(Reflectable<std::decay_t<T>>)
 constexpr std::size_t field_count_of()
 {
-    return reflection_detail::FIELD_NAMES<std::decay_t<T>>.size();
+    return fixed_containers::reflection::customize::ReflectionHelper<std::decay_t<T>>::FIELD_NAMES
+        .size();
 }
 
 template <typename T>
     requires(Reflectable<std::decay_t<T>>)
 constexpr const auto& field_names_of()
 {
-    return reflection_detail::FIELD_NAMES<std::decay_t<T>>;
+    return fixed_containers::reflection::customize::ReflectionHelper<std::decay_t<T>>::FIELD_NAMES;
 }
 
 template <typename T, typename Func>
     requires(Reflectable<std::decay_t<T>>)
 constexpr void for_each_field(T&& instance, Func&& func)
 {
-    constexpr const auto& FIELD_NAMES = field_names_of<T>();
-    auto tuple_view = tuples::as_tuple_view<FIELD_NAMES.size()>(instance);
-    tuples::for_each_entry(tuple_view,
-                           [&func]<typename Field>(std::size_t index, Field&& field)
-                           { func(FIELD_NAMES.at(index), std::forward<Field>(field)); });
+    return fixed_containers::reflection::customize::ReflectionHelper<std::decay_t<T>>::apply(
+        std::forward<T>(instance), std::forward<Func>(func));
```

Comment:
why call it `apply` instead of `for_each_field`?

Reply:
Changed to `for_each_field()`